Render fieldcompare diff visualizations (#883)
precice/tutorials: tools/tests/README.md

- `precice-exports/`: The coupling meshes of the test run.
- `reference-results/`: The coupling meshes of the reference run, as stored on Git LFS, expanded into `reference-results-unpacked`. For test cases using implicit coupling, the reference `.tar.gz` also contains the reference `precice-*-iterations.log` files.

> - `diff-results/`: Numerical difference of the results in the two directories (computed with `fieldcompare dir --diff precice-exports/ reference/`). These are only present on failed comparisons. The `visualizations/` subdirectory contains one PNG per numeric point field, rendered as sphere glyphs and colored by the difference values (via the `diff_visualizer` Docker stage using OSMesa, so no display server is required).

Reviewer: I would avoid implementation details in the user documentation. These are meant either for code comments or for the PR description.

```suggestion
- `diff-results/`: Numerical difference of the results in the two directories (computed with `fieldcompare dir --diff precice-exports/ reference/`). These are only present on failed comparisons and accompanied by visualization in `diff-results/visualization/`.
```

`system_tests_run_<run_id>_<run_attempt>_full`

> (a smaller `_logs` archive contains the stage log files and, on comparison failures, the PNG diff visualizations under `runs/*/diff-results/visualizations/`). The archive contains a shared `runs/` directory:

Reviewer: It is clear where to find these once one looks at the archive:

```suggestion
(a smaller `_logs` archive contains the stage log files and, on comparison failures, difference visualizations. The archive contains a shared `runs/` directory:
```
